proposals/0234-proxy-rpc-generation.md

```diff
@@ -12,17 +12,52 @@ This proposal adds automatic RPC generation for iOS and Java from the XML spec v
 Adding RPCs is currently difficult. Every RPC to be added requires handwriting each parameter, then review, and then unit testing. This makes large RPC changes very difficult, and at times, error-prone. Beyond that, alterations to the RPC classes require an enormous amount of work due to the number of RPCs that already exist – in addition to the problems of code duplication. This does not happen often, but it does happen.
 
 ## Proposed solution
-The proposed solution is to create python scripts that auto-generate Objective-C, Android and JavaScript code based on a given SDL MOBILE_API XML spec. This allows us to generate new classes whenever the XML spec changes, and to update the script whenever the classes themselves need to change.
+The proposed solution is to create python scripts that auto-generate Markdown, C++, Objective-C, Android/Java and JavaScript code based on a given SDL MOBILE_API XML spec. This allows us to generate new content whenever the XML spec changes, and to update the script whenever the classes themselves need to change.
 
 The script should take a `MOBILE_API` XML spec as input and a location to output. The script will then output the RPC files at that location.
 
 ### Implementation Notes
-1. The spec is not currently housed within the library. Therefore the `rpc_spec` repository will be pulled into each library repository as a submodule. The `MOBILE_API` in the `rpc_spec` submodule will be the version used by the script to output the RPC files.
-2. This proposal is left intentionally ambiguous regarding implementation details of the scripts. I wish to leave implementation details up to the author(s) of the scripts. However, the command-line switches should be identical between implementations.
+1. The spec is not currently housed within the library. Therefore the `rpc_spec` repository will be pulled into each library repository as a submodule. The `MOBILE_API` in the `rpc_spec` submodule will be the version used by the script to output the RPC files. Following repositories should include `rpc_spec`:
+
+    `sdl_core`, `sdl_java_suite`, `sdl_ios`, `sdl_javascript_suite`
+
+2. The command-line switches should be identical between implementations. See "Command-line switches" section for details.
```

Comment:
Based on [this comment](https://github.com/smartdevicelink/sdl_evolution/issues/853#issuecomment-548346897), I believe the following language should still be included in this section: This proposal is left intentionally ambiguous regarding implementation details of the scripts. I wish to leave implementation details up to the author(s) of the scripts.